build.sbt

```diff
@@ -110,12 +107,19 @@ lazy val `kyo-scheduler-zio` =
                 "-Wunused:all",
                 "-language:strictEquality"
             ),
-            scalacOptions += "-Xsource:3",
             libraryDependencies += "dev.zio"       %%% "zio"       % zioVersion,
-            libraryDependencies += "org.scalatest" %%% "scalatest" % "3.2.16" % Test,
-            crossScalaVersions                      := List(scala3Version, scala212Version, scala213Version)
+            libraryDependencies += "org.scalatest" %%% "scalatest" % "3.2.16" % Test
         )
 
+lazy val `kyo-scheduler-zio-3` = `kyo-scheduler-zio-base`.settings(
+    crossScalaVersions := List(scala3Version)
+)
+
+lazy val `kyo-scheduler-zio` = `kyo-scheduler-zio-base`.settings(
+    scalacOptions += "-Xsource:3",
+    crossScalaVersions := List(scala3Version, scala212Version, scala213Version)
+)
```

Comment:
I think this should be possible without separate values with `++= if (_) Seq("-Xsource:3") else Nil`

Reply:
Yes, we can exclude Scala 3 here

Reply:
done